dev/io.openliberty.microprofile.config.2.0.internal_fat/fat/src/io/openliberty/microprofile/config/internal_fat/Config20Tests.java

```diff
@@ -41,22 +42,34 @@
 @RunWith(FATRunner.class)
 public class Config20Tests extends FATServletClient {
 
-    public static final String DEFAULT_SOURCES_APP_NAME = "defaultSourcesApp";
     public static final String CLASS_LOADER_APP_NAME = "classLoadersApp";
     public static final String CONVERTER_LOADER_APP_NAME = "convertersApp";
+    public static final String DEFAULT_SOURCES_APP_NAME = "defaultSourcesApp";
+
+    // new to mpConfig-2.0
+    public static final String CONFIG_PROPERTIES_APP_NAME = "configPropertiesApp";
     public static final String PROPERTY_EXPRESSION_APP_NAME = "propertyExpressionApp";
     public static final String UNWRAP_APP_NAME = "unwrapApp";
 
+    // Config Property values for ConfigProperties tests
+    public static final String NO_PREFIX_TEST_KEY = "validPrefix.validkey";
+    public static final String NO_PREFIX_TEST_VALUE = "value";
+    public static final String CAMEL_CASE_TEST_KEY = "validPrefix.validCamelCaseKey";
+    public static final String CAMEL_CASE_TEST_VALUE = "aValueFromCamelCase";
```

Comment:
These might be better on the test servlet they're used for rather than in this central test class.